syntax = "proto3";

package greeter;

// The greeter service definition
service Greeter {
  rpc SayHello (HelloRequest) returns (HelloResponse);
}

// The request message containing the user's name
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloResponse {
  string message = 1;
}
